### PR TITLE
chore: update version for npm release

### PR DESCRIPTION
**Pull request by andoulla**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
     "npm-run-all": "^4.1.5",
     "parcel-bundler": "^1.12.4",
     "prettier": "^2.2.1",
-    "react": "17.0.1",
+    "react": "^16.8.0 || ^17",
```
Comment (thomasdigby) [inline package.json:87]: What's the reason for this change? This is a dev dependency so shouldn't affect how it's used in other projects

Comment (andoulla) [inline package.json:87]:
```suggestion
    "react": "17.0.1",
```

Comment (andoulla) [reply package.json]: I have reverted it, it randomly comes up and we don't know what's triggering it, to be honest. we had this problem last week

**Review by thomasdigby — APPROVED** [review]
